### DISCUSSION TITLE
Iterate in a StandartTableView rows declaration

### DISCUSSION THREAD

**Discussion by guillaumeAbel** [slint-ui/slint#4643] [Q&A] [answered]
Hi everyone,

I have to set a lot of element in a table, so i want to iterate in the declarations of the rows attribute, but i don't even know if it's possible.
Currently i tried this syntax:
```
export component App inherits Window {
    width: 900px;
    height: 900px;
    table:= StandardTableView {
        width: 100%;
        height: 100%;
        // Define columns
        columns: [
            { title: "Name" },
            { title: "Value" }
        ];
        // Bind rows to the data property
        for row in 50000 : rows {
            [{text: "col1"}, {text: "col2"}]
        }
    }
}
```
but this not compile with parse errors on the square brackets and the curly.

Thanks for help

**Comment by ogoffart** [comment, marked as answer]
You have to do fill your model from the native code.
For example, in rust: https://github.com/slint-ui/slint/blob/c399447c932dc74be3165a8f670285d4671e5845/examples/gallery/main.rs#L26-L38

**Reply by guillaumeAbel** [reply]
Thanks ! 👍

**Reply by chlghdwns** [reply]
If you succeeded in implementing it, I wonder what you did.
